sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/stringExpressions.scala

```diff
@@ -644,6 +652,7 @@ object StringTrimLeft {
       > SELECT _FUNC_('Sp', 'SSparkSQLS');
        arkSQLS
   """)
+// scalastyle:on line.size.limit
```

Comment:
Looks we are okay here with disabling.

Reply:
Yeah, I see. Just put it in case. Since you said, I will remove it. :)